lib/src/DateUtils.dart

```diff
@@ -264,12 +264,12 @@ class DateUtils {
     return now;
   }
 
-  static DateTime _parseNextLast(String s, DateTime now, {bool last = false}) {
+  static DateTime? _parseNextLast(String s, DateTime now, {bool last = false}) {
```

Comment:
We can change this into the Non-Nullable method by changing the default value as `DateTime.now()` instead of `null` https://github.com/Ephenodrom/Dart-Basic-Utils/blob/ce38a41d074c25f777d2a0c18f30617bb587a68c/lib/src/DateUtils.dart#L327

@Ephenodrom  what do you think about it?

Reply:
Yes ```DateTime.now()``` seems to be the best solution here.